### PR TITLE
Publish ychaos images on each release

### PR DESCRIPTION
**Pull request by vijaybabu4589**
# Summary

@yahoo/ychaos-dev

Publish Python version specific YChaos images on each release

---
SD Flow

image

YChaos Doc changes

image

#### Prerequisites
- [x] I have read the contribution guidelines

#### Code Analysis
- [ ] Covered by Unittests
- [ ] Security warnings ignored (Why?!)

#### Project related
- [ ] Schema changed and is backward compatible.
- [ ] Introduces a new sub-command for ychaos CLI

### Autogenerated Files
- [ ] Auto generated schema

### Pre-Merge Checklist

- [x] All the build jobs are passing

---

### Checklist (Reviewer 1)

- [ ] Reviewed Source Code
- [ ] Reviewed Tests (If applicable)
- [ ] Reviewed Documentation (If applicable)

### Checklist (Reviewer 2)

- [ ] Reviewed Source Code
- [ ] Reviewed Tests (If applicable)
- [ ] Reviewed Documentation (If applicable)

### CODE REVIEW

File: screwdriver.yaml [hunk 1]
```diff
@@ -224,6 +207,75 @@ jobs:
             PUBLISH: True
         requires: [version_pypi]
 
+    # Publish YChaos image with Python 3.6 and make this image as default Latest image
+    publish_docker_PY36:
```
Comment (shashankrnr32) [inline screwdriver.yaml:211]:
```suggestion
    publish_docker_py36:
```

File: screwdriver.yaml [hunk 2]
```diff
@@ -224,6 +207,75 @@ jobs:
...
+            PY_VERSION: 3.6
+        requires: [package_publish_pypi]
+        steps:
+            - login: *docker_login
+            - build: *docker_build
+            - latest_tag: docker tag $DOCKER_REPO:py$PY_VERSION-latest $DOCKER_REPO:latest
+            - publish: *docker_publish
+
+    # Publish YChaos image with Python 3.7
+    publish_docker_PY37:
```
Comment (shashankrnr32) [inline screwdriver.yaml:229]:
```suggestion
    publish_docker_py37:
```

File: screwdriver.yaml [hunk 3]
```diff
@@ -224,6 +207,75 @@ jobs:
...
+            DOCKER_REPO: ychaos/ychaos
+            PY_VERSION: 3.7
+        requires: [package_publish_pypi]
+        steps:
+            - login: *docker_login
+            - build: *docker_build
+            - publish: *docker_publish
+
+    # Publish YChaos image with Python 3.8
+    publish_docker_PY38:
```
Comment (shashankrnr32) [inline screwdriver.yaml:246]:
```suggestion
    publish_docker_py38:
```

File: screwdriver.yaml [hunk 4]
```diff
@@ -224,6 +207,75 @@ jobs:
...
+            DOCKER_REPO: ychaos/ychaos
+            PY_VERSION: 3.8
+        requires: [package_publish_pypi]
+        steps:
+            - login: *docker_login
+            - build: *docker_build
+            - publish: *docker_publish
+
+    # Publish YChaos image with Python 3.9
+    publish_docker_PY39:
```
Comment (shashankrnr32) [inline screwdriver.yaml:263]:
```suggestion
    publish_docker_py39:
```

**Review by shashankrnr32 — COMMENTED** [review]
nit: Keep the job names consistent with test_validation_py36

**Comment by shashankrnr32** [comment]
This is a great addition to the project. Thanks @vijaybabu4589

**Review by shashankrnr32 — APPROVED** [review]
👍

**Review by AlfinST — APPROVED** [review]
🚀

**Review by sushilkar — APPROVED** [review]

**Pull request merged**